shell/common/rasterizer.h

```diff
@@ -8,6 +8,7 @@
 #include <memory>
 #include <optional>
 
+#include "display_list/display_list_image.h"
```

Comment:
The header file references here and elsewhere are a bit incorrect, I'll fix them later